modules/grafana/values/grafana-values.yaml.tpl

```diff
@@ -0,0 +1,63 @@
+persistence:
+  enabled: true
+  type: pvc
+  size: 10Gi
+
+ingress:
+  enabled: true
+  annotations:
```

Comment:
we still need this annotations to be configurable(can be with this default),  in some places we use nginx ingress instead of alb